src/Microsoft.AspNetCore.Testing/project.json

```diff
@@ -24,6 +23,7 @@
     },
     "dnxcore50": {
```

Comment:
It has to reference the runner for the conditional test attribute to be discovered.